src/reusables.js

```diff
@@ -95,7 +95,7 @@ export function loadTextFile(file, func) {
 }
 
 export function addFileProtocol(path) {
-    return path.startsWith("/") || path.startsWith("./")
+    return path.startsWith("/") || path.startsWith("./") || path.startsWith("C")
```

Comment:
I forgot this code ! 
It's a good fix, but I think I will change this code part later because it's really too specific.

Reply:
No problem thanks for implementing the changes. Not sure what you had in mind for new version, but one suggestion is getting the root from [path.parse.](https://nodejs.org/api/path.html#path_path_parse_path)

Looking forward to seeing the new version!

Reply:
I can't touch this part until a breaking 2.0.0.
Indeed the use of path.parse could be critical for the browser usage of the library.

In a 2.0.0 I hope to split browser and serveur usage and thus clarify this point.